### PR TITLE
ROX-18008: Delete non-postgres code in Entity UI

### PR DESCRIPTION
**Pull request by pedrottimark**
## Description

> deprecating our Postgres feature flag

Van, can you please look carefully at changes to GraphQL queries?

### Analysis

Find in Files decrease from 36 results in 30 files to 24 results in 18 files

| Folder                        | results | files |
| :---                          |    ---: |  ---: |
| Components/workflow           |       1 |     1 |
| Containers/VulnMgmt/Dashboard |       1 |     1 |
| Containers/VulnMgmt/Entity    |      12 |    12 |
| Containers/VulnMgmt/List      |      10 |     8 |
| Containers/VulnMgmt/widgets   |      11 |     7 |
| types                         |       1 |     1 |
|                               |      36 |    30 |

### Residue

1. Fix inconsistent case in phrase under **Findings**
2. Investigate effort to replace **the whatever you are looking for cannot be found** with 404 page for obsolete entity types: components, cves, and policies.
3. Investigate whether VulnMgmtPolicyQueryUtil is still relevant.
4. Verify whether deletions from List folder remove last reference:
    * VULN_CVE_LIST_FRAGMENT

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

## Testing Performed

1. `yarn lint` in ui
2. `yarn build` in ui
    * `wc build/static/js/*.js`
        branch - master: -13771 = 11771992 - 11785763
3. `yarn start` in ui

### Manual testing

1. Visit /main/vulnerability-management/clusters and click a row
    * See 3 tabs under **Cluster Findings**
        clusters

2. Visit /main/vulnerability-management/deployments and click a row that has fixable CVEs
    * See fixable image CVEs in table under **Deployment Findings**
        deployments

3. Visit /main/vulnerability-management/images and click a row that has fixable CVEs
    * See fixable image CVEs in table under **Image Findings**
        images

4. Visit /main/vulnerability-management/namespaces and click a row that has fixable CVEs
    * See fixable image CVEs in table under **Namespace Findings**
        namespaces

5. Visit /main/vulnerability-management/nodes and click a row that has fixable CVEs
    * See fixable node CVEs in table under **Node Findings**
        nodes

### Integration testing

1. `yarn cypress-open` in ui/apps/platform
    * vulnmanagement/clusterCves.test.js
    * vulnmanagement/clusters.test.js
    * vulnmanagement/deployments.test.js
    * vulnmanagement/imageComponents.test.js
    * vulnmanagement/imageCves.test.js
    * vulnmanagement/images.test.js
    * vulnmanagement/namespaces.test.js
    * vulnmanagement/nodeComponents.test.js
    * vulnmanagement/nodeCves.test.js
    * vulnmanagement/nodes.test.js

### CODE REVIEW

File: ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js [hunk 1]
```diff
@@ -45,16 +44,15 @@ const TableWidgetFixableCves = ({
     const [cveSort, setCveSort] = useState(defaultCveSort);
 
     const { isFeatureFlagEnabled } = useFeatureFlags();
-    const showVMUpdates = isFeatureFlagEnabled('ROX_POSTGRES_DATASTORE');
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js:48]: I think we can also remove `useFeatureFlags()` completely from this file too ?

File: ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js [hunk 2]
```diff
@@ -45,16 +44,15 @@ const TableWidgetFixableCves = ({
...
     const queryFieldName = queryFieldNames[entityType];
-    let queryVulnCounterFieldName = showVMUpdates ? 'imageVulnerabilityCounter' : 'vulnCounter';
-    let queryVulnsFieldName = showVMUpdates ? 'imageVulnerabilities' : 'vulns';
-    let queryCVEFieldsName = showVMUpdates ? 'imageCVEFields' : 'cveFields';
-    let queryFragment = showVMUpdates ? IMAGE_CVE_LIST_FRAGMENT : VULN_CVE_LIST_FRAGMENT;
+    let queryVulnCounterFieldName = 'imageVulnerabilityCounter';
+    let queryVulnsFieldName = 'imageVulnerabilities';
+    let queryCVEFieldsName = 'imageCVEFields';
+    let queryFragment = IMAGE_CVE_LIST_FRAGMENT;
     let exportType = entityTypes.CVE;
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js:56]: I think we can completely remove the export type of `entityTypes.CVE` from this section, too.

**Review by vjwilson — APPROVED** [review]
I noticed a couple of places where a little more legacy code could be removed, but those could be addressed either now, or in a follow-up PR.

File: ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js (hunk 1 above)
Comment (pedrottimark) [reply ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js:48]: Good question. It is still used indirectly:

1. `getCveTableColumns(workflowState, isFeatureFlagEnabled)`
2. `getFilteredCVEColumns(columns, workflowState, isFeatureFlagEnabled)`
3. `isFeatureFlagEnabled('ROX_ACTIVE_VULN_MGMT')`

**Comment by pedrottimark** [comment]
ocp-4-10-ui-e2e-tests failure because pod restarts: collector

**Comment by pedrottimark** [comment]
gke-ui-e2e-tests failure

> Workload table toolbar should correctly handle applied filters

> Timed out retrying after 4000ms: Expected to find element: `.pf-c-toolbar .pf-c-chip-group *:contains("Namespace") + ul li:contains("stackrox")`, but never found it.

**Pull request merged**